RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+19.4
+-----
+* [**] Adds animated Gif support in notifications and comments [#17981]
```

Comment:
I see that this PR is pointing to 19.3 -- just a reminder that this needs to go into the 19.3 section